index.html

```diff
@@ -41,5 +41,142 @@ <h1 class="header-text">Hey there. I’m Raphael
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+
+            <div class="card-item">
+                
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+
+            <div class="card-item">
+               
+                <div class="down-card">
+                    <p>Multi-Post Stories Gain+Glory</p>
+                    <div class="skills"><div>Ruby on rails</div><div>CSS</div><div>JavScript</div><div>html</div></div>
+                    <div class="button"><p>See Project</p></div>
+                </div>    
+            </div>
+        </div>
+
+
+        <section>
+            <div class="about-me">
+                <h3>About me</h3>
+                <div class="about-me-text">
+                 <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="my-resume"><p>Get My Resume</p></div>
+            </div>
+        </section>
+
+        <section>
+            <div class="skills-container">
+                <div class="skill-item">
+                    <img src="img/icon-languages.svg" alt="Language">
+                    <p>Language</p>
```

Comment:
This is supposed to be a heading and as such, I suggest using a header tag (preferably an `h3` tag) by so doing, you will have a more semantic code and well-ranked page content.